src/container/ModalDialog/ModalDialog.tsx

```diff
@@ -17,12 +17,12 @@ const ModalDialog: FC<ModalDialogProps> = (props) => {
   return (
     <ModalBackground>
       <div className='w-4/5 h-auto lg:w-1/2 xl:w-2/5 xl:max-w-screen-sm bg-white text-center rounded-lg flex flex-col mx-5 dark:bg-gray-800 dark:text-white'>
-        <div className='flex'>
+        <div className='flex pr-1'>
```

Comment:
Looks nice, but i think it would look even better with 

```suggestion
        <div className='flex pr-2 pt-1'>
```